### PR TITLE
Update to readme for kudo init.  Adding new cli commands

### PR DESCRIPTION
**Pull request by kensipe**
**What this PR does / why we need it**:

Updates to README / Getting Started
* Adding details on kudo init

Adding new CLI Commands to CLI docs
Organizing CLI commands in alphabetical order

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by kensipe** [comment]
This is work in-progress...  Looking to add more rich details for CLI commands 

kudo init
kudo repo index
kudo repo add|remove|context
flag:  `--repo` for install and upgrade

**Comment by kensipe** [comment]
ready for review

File: content/docs/README.md [hunk 1]
```diff
@@ -2,30 +2,42 @@
...
+
+## Install KUDO into your cluster
+
+Once you have a running cluster with `kubectl` installed along with the KUDO CLI plugin, you can install KUDO like so:
+
+```
+kubectl kudo init
+```
+
+If you want to manage the installation by hand the follow is also possible:
```
Comment (yankcrime) [inline content/docs/README.md]: follow → following

**Review by gerred — APPROVED** [review]

File: content/docs/README.md [hunk 2]
```diff
@@ -2,35 +2,62 @@
...
 
 If you plan on developing and testing KUDO locally via Minikube, you'll need to launch your cluster with a reasonable amount of memory allocated. By default, Minikube runs with 2GB - we recommend at least 10GB, especially if you're working with applications such as [Kafka](examples/apache-kafka.md). You can start Minikube with some suitable resource adjustments as follows:
 
 ```bash
 minikube start --cpus=4 --memory=10240 --disk-size=40g
 ```
+
+## Notes on KIND
+
+In order to use KIND with storage operators, it is necessary to modify it's Persistence Storage ([more details](https://dischord.org/2019/07/11/persistent-storage-kind/)).
```
Comment (yankcrime) [inline content/docs/README.md]: it's → its
Persistence → Persistent

**Pull request merged**